File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        my_set = set(nums)

        max_len = 0

        for num in nums:
            if num-1 not in my_set:
                curr_num = num
                length = 1
                while curr_num + length in my_set:
                    length +=1
                max_len = max(length,max_len)

        return max_len
        

File: Data Structures & Algorithms/duplicate-integer/submission-1.cpp
#include <set>

class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {

        set <int> mySet;

        for(int i = 0; i < nums.size(); i++){
            mySet.insert(nums.at(i));
        }

        if(mySet.size() < nums.size()){
            return true;
        }
        else{
            return false;
        }
    }
};
